containers/graalvm-build/docker/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM ubuntu:20.04
+
+# A build container for Graal project.
+
+LABEL description="R3 GraalVM build container"
+LABEL maintainer="conclave@r3.com"
+ARG commit_id
+LABEL commit_id=${commit_id}
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y && apt-get install -y \
+    git \
+    curl \
+    apt-transport-https \
+    ca-certificates \
+    software-properties-common \
+    gnupg \
+    wget
+
+RUN apt-get update -y && apt-get install -y \
```

Comment:
Why is this split into two? Or a comment explaining why. Could it be because this second update is more likely to change than the first so benefit from fewer changes in the image layers?